### PR TITLE
List Madbyte (MBYT)

### PR DESCRIPTION
**Pull request by madbytecoin**
- Official block explorer URL: http://explorer.madbyte.com:3001/
- altcoin (based on Litecoin)
- Official project URL: http://madbyte.com

### CODE REVIEW

File: src/main/java/bisq/core/trade/statistics/TradeStatisticsManager.java [hunk 1]
```diff
@@ -306,6 +306,8 @@ private void printAllCurrencyStats() {
         newlyAdded.add("GEO");
         newlyAdded.add("GRANS");
         newlyAdded.add("ICH");
+        newlyAdded.add("MBYT");
```
Comment (blabno) [inline src/main/java/bisq/core/trade/statistics/TradeStatisticsManager.java]: Do not touch this file.

File: src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 2]
```diff
@@ -133,6 +133,7 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         result.add(new CryptoCurrency("GRC", "Gridcoin"));
         if (!baseCurrencyCode.equals("LTC"))
             result.add(new CryptoCurrency("LTC", "Litecoin"));
+        result.add(new CryptoCurrency("MBYT", "Madbyte"));
```
Comment (blabno) [inline src/main/java/bisq/core/locale/CurrencyUtil.java]: Do not touch this file.

File: src/test/java/bisq/core/payment/validation/AltCoinAddressValidatorTest.java [hunk 3]
```diff
@@ -988,6 +988,21 @@ public void testPHR() {
         assertFalse(validator.validate("").isValid);
     }
 
+    @Test
```
Comment (blabno) [inline src/test/java/bisq/core/payment/validation/AltCoinAddressValidatorTest.java]: @cbeams do we want anybody to touch this file anymore?

**Comment by cbeams** [comment]
Instructions for how to list an asset are being changed as we type. Stay tuned for further updates.

**Comment by cbeams** [comment]
The process for listing an asset has recently changed (see #32), and I am now closing this pull request along with all other outstanding asset listing pull requests.

Please read and follow the new instructions at https://bisq.network/list-asset and re-submit a new pull request when ready. Note that Bisq v0.7.0 will likely ship within the next week, so you should still be able to get your asset listed in time for that release. Thanks.

**Pull request closed without merging**